cassandane/doc/running_tests.txt

```diff
@@ -16,146 +16,103 @@ Cassandane tests are run out of the Cassandane directory itself, without
 installing Cassandane anywhere.  This is not the result of deliberate policy so
 much as implementation laziness.
 
-All runtime state is created under the directory /var/tmp/cass/, and
-results reports are written to the current working directory.  Apart from
-reports, Cassandane does not require write permission to its source directory.
+All runtime state is created under the cassandane rootdir configured in
+cassandane.ini (by default: /var/tmp/cass).
 
-Cassandane (or more precisely, the Cyrus code it exercises) needs to be run
-either as the superuser or as the "cyrus" user.  If run as another user,
-it attempts to use 'sudo' to re-run itself as the "cyrus" user.
+Internally, Cassandane (or more precisely, the Cyrus code it exercises) needs
+to be run either as the superuser or as the "cyrus" user.  But you should
+generally invoke Cassandane as yourself, not as "cyrus" or "root".  It will
+to re-run itself using sudo, which you already configured during setup (didn't
```

Comment:
That sentence doesn't scan for me - should that "to" be a "then"?

Reply:
"it will _try_ to", doh, thanks!

Reply:
fixed and force-pushed :)